5.sc_Morphology_Logistic_Regression/1.train_models/0.train_single_class.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    df_nomic = df_nomic.drop(columns=df_nomic.columns[0:2])
</pre><p>Consider refactoring these two lines to:</p><pre class="ql-syntax" spellcheck="false">df_nomic = df_nomic.drop(columns=df_nomic.columns[0:25])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>python list slicing isn't inclusive at the end. In this case I want to keep column 2</p>